cla_public/templates/macros/subform.html

```diff
@@ -42,7 +42,8 @@
       <li>{{ _('If your nominated person misses the call you’ll need to complete this form again.') }}</li>
     </ul>
     {{ Form.group(form.thirdparty.full_name, field_attrs={'class': 'm-large'}) }}
-    {{ Form.group(form.thirdparty.relationship) }}
+    {{ Form.group(form.thirdparty.relationship,
+      field_attrs={'class': ''}) }}
```

Comment:
Can we remove the linespace so it's on one line? Just so it's a bit more readable 😅